src/Uno.UI/Helpers/PlatformRuntimeHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uno.UI.Helpers;
+
+internal static class PlatformRuntimeHelper
+{
+	internal static UnoRuntimePlatform SkiaPlatform { get; set; } = UnoRuntimePlatform.Skia;
+
+	public static UnoRuntimePlatform Current => GetPlatform();
+
+	private static UnoRuntimePlatform GetPlatform() =>
+#if __ANDROID__
+		UnoRuntimePlatform.Android;
+#elif __IOS__
+		UnoRuntimePlatform.iOS;
+#elif __MACCATALYST__
+		UnoRuntimePlatform.MacCatalyst;
+#elif __MACOS__
+		UnoRuntimePlatform.MacOSX;
+#elif __WASM__
+		UnoRuntimePlatform.WebAssembly;
+#elif __SKIA__
+		SkiaPlatform;
+#elif !HAS_UNO
+		UnoRuntimePlatform.Windows;
```

Comment:
This feels like a dead code?

Reply:
What other conditional could I use?

Reply:
For Windows, you don't need it here. It should be handled by code in Uno.UI.Toolkit